README.md

```diff
@@ -34,33 +34,28 @@ export base64_capture_macs_script_content=$(cat capture-macs.sh|base64 -w 0) &&
 fcct custom-config.fcc > file.ign
 ```
 
-  - Upload `file.ign` to a shared location which the OpenShift nodes can access.
+  - Upload `file.ign` to a shared location where the OpenShift nodes can access.
 
-- Base64 encode the `setup-ovs.sh` file and paste the content into each MCO file into "base64_script_content" section. 
+- Base64 encode the `init-interfaces.sh` file and paste the content into each MCO file into "base64_script_content" section. 
 
     **TIP:** To update the content you can use:
 
 ```
-export base64_script_content=$(cat setup-ovs.sh|base64 -w 0) && envsubst <  mco_ovs_workers.yml.tmpl > mco_ovs_workers.yml && envsubst < mco_ovs_supervisor.yml.tmpl > mco_ovs_supervisor.yml
+export base64_script_content=$(cat init-interfaces.sh|base64 -w 0) && envsubst <  mco_ovs_workers.yml.tmpl > mco_ovs_workers.yml && envsubst < mco_ovs_supervisor.yml.tmpl > mco_ovs_supervisor.yml
 ```
 
-- MCO files are MachineConfig, which you can apply manually once the cluster is up or add it to the installation automation/pipeline.
-
 ### 3. Run the installation
 - Follow the guide to install a bare-metal cluster in the [OpenShift production documentation.
 ](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/installing/installing-on-bare-metal)
 
 ## Notes
-- Changing network configuration is done with the `setup-ovs.sh` file. 
-
-- In order to change the bond type to something other than balance-slb (for example active-backup) ,open the  `setup-ovs.sh` file and search for the “#make bond” section. 
-Change *“ovs-port.bond-mode”* to the desired type and make sure that all other related settings are aligned. 
+- Basic network configuration is set with the `init-interfaces.sh` file, which run via MCO files. MCO files are MachineConfig, which you can apply manually once the cluster is up or add it to the installation automation/pipeline. MachineConfig file is a "Day 2" tool that allows to configure or run scripts on a machine with an installed OS (post-installation).
```

Comment:
Not sure we need to need to combine both paragraph's, too much words people wont read

Reply:
I feel it is better placed here since this is where the MCO comes into play.

Reply:
Agree that it comes into play, but we explaining about MCO later the in the paragraph. 
In the first line your saying what your doing and then your explaining, I think it should be the other way around, first you explain about MCO then what your doing with it